includes/import/class-wc-product-csv-importer.php

```diff
@@ -384,7 +384,7 @@ public function parse_tax_status_field( $value ) {
 	 *
 	 * @param string $value Field value.
 	 *
-	 * @return array of arrays with "parent" and "name" keys.
+	 * @return array|WP_Error
```

Comment:
@juliaamosova The function doesn't return a `WP_Error` on any flow.

Reply:
Thanks @rrennick! This was added by PhpStorm automatically when I was editing the code. My bad for not catching this during commit. Fixed.